google/cloud/bigtable/examples/opencensus/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenCensus example for GCP Bigtable C++ Client.
+## Before you begin
+### About OpenCensus
+[OpenCensus](https://opencensus.io) is a vendor-agnostic single distribution of libraries to provide metrics collection and tracing for your services. In this example we show how one can integrate OpenCensus with the Cloud Bigtable C++ client library.
+
+### Setup Stackdriver
+The OpenCensus example will publish metrics and traces to [Stackdriver](https://cloud.google.com/stackdriver/), if you have not previously configured Stackdriver, follow the instructions in [...](https://github.com/census-instrumentation/opencensus-cpp/tree/master/examples/grpc#stackdriver) to enable Stackdriver for your Google Cloud project.
```

Comment:
One last nit, change that `follow the instructions in [...]` to maybe `follow [these instructions]`?